pkg/skaffold/yamlutil/yaml.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2019 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package yamlutil
+
+import (
+	"bytes"
+	"io"
+
+	yaml "gopkg.in/yaml.v3"
+)
+
+func UnmarshalStrict(in []byte, out interface{}) error {
```

Comment:
Public methods should get comment sentences: https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences